src/main/java/Bosco.java

```diff
@@ -0,0 +1,96 @@
+import java.sql.SQLOutput;
+import java.util.Scanner;
+
+public class Bosco {
+    private static final String DIVIDER = "\t____________________________________________________________";
+    private static final int MAX_TASKS = 100;
```

Comment:
I like how you used constants to define the literals that are used in your code! :)